stake-pool/program/src/instruction.rs

```diff
@@ -777,7 +777,7 @@ pub fn remove_validator_from_pool(
         AccountMeta::new_readonly(*stake_pool_withdraw, false),
         AccountMeta::new(*validator_list, false),
         AccountMeta::new(*stake_account, false),
-        AccountMeta::new_readonly(*transient_stake_account, false),
+        AccountMeta::new(*transient_stake_account, false),
```

Comment:
i cant figure out how comment or suggest corrections on lines github considers out of scope but line 115 should change to `///   5. `[w]` Transient stake account`

Reply:
Ah great point, will fix, thanks!